### PR TITLE
add update hint component

### PR DESCRIPTION
**Pull request by TMysiv**

### CODE REVIEW

File: src/components/hint/updateHint/UpdateHint.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+              id='info'
+              label='Info'
+              value={formik.values.info}
+              onChange={formik.handleChange}
+              onBlur={formik.handleBlur}
+              helperText={formik.touched.info && formik.errors.info}
+              FormHelperTextProps={{ style: { color: 'red', fontSize: '11px' } }} />
+            <FileUpload
+              multiple
+              callback={fileBase64 => {
```
Comment (RamboUA) [inline src/components/hint/updateHint/UpdateHint.tsx:86]: Can we write it as a function outside a JSX?

File: src/components/hint/updateHint/UpdateHint.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          navigate('/hints')
+        }
+      } catch (err: any) {
+        setError(err.response.data.message);
+      } finally {
+        setImg('');
+        setImgErr(false);
+      }
+    },
+    validationSchema: yup.object({
```
Comment (RamboUA) [inline src/components/hint/updateHint/UpdateHint.tsx:56]: I would recommend to keep initial values and validation schema in a module nearby. You could call it something like config.js in updateHint folder

**Review by RamboUA — CHANGES_REQUESTED** [review]
Nice. Just consider please a couple of comments:

File: src/components/hint/updateHint/UpdateHint.tsx (hunk 1 above)
Comment (TMysiv) [reply src/components/hint/updateHint/UpdateHint.tsx:86]: fix

File: src/components/hint/updateHint/UpdateHint.tsx (hunk 2 above)
Comment (TMysiv) [reply src/components/hint/updateHint/UpdateHint.tsx:56]: fix

**Review by RamboUA — APPROVED** [review]
Good thank you

**Pull request merged**